WTG.Analyzers/Analyzers/Visibility/VisibilityAnalyzer.cs

```diff
@@ -41,7 +43,11 @@ static void Analyze(SyntaxNodeAnalysisContext context, FileDetailCache cache)
 				switch (kind)
 				{
 					case SyntaxKind.PrivateKeyword:
-						context.ReportDiagnostic(Rules.CreateDoNotUseThePrivateKeywordDiagnostic(modifier.GetLocation()));
+						privateLocation = modifier.GetLocation();
```

Comment:
Perhaps we should just store the private modifier object, so that we don't call `GetLocation()` in the case of private-protected methods? Or is `GetLocation()` so cheap as to not be worth it?

Granted, I doubt we will encounter many `private protected` things in practice...

Reply:
It only allocates an object, but on the other hand, it allocates an object :)
Copying the struct would almost certainly be cheaper in the odd case where we use private protected.